src/star-ratings.js

```diff
@@ -77,12 +77,11 @@ class StarRatings extends React.Component {
     return offsetValue;
   }
 
-  get fillId() {
+  fillId =
     // Using the rating's decimal place to set a unique id for the fill. e.g. a rating of 3.5 and 4.5 will use the same half fill star with id #star-grad-5.
```

Comment:
move comment above the variable declaration.

Reply:
whoops